projects/worker-basic/simple-test/params.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
The tests for `worker-basic` are not in the right folder, hence they aren't picked up: https://travis-ci.com/github/adobe/asset-compute-example-workers/jobs/338574958 (`No test cases found for worker in /home/travis/build/adobe/asset-compute-example-workers/projects/worker-basic`).

They must be in a folder `tests`, e.g. `projects/worker-basic/tests/simple-test/param.json`.

BTW, we might rename the folder after discussion with the aio team yesterday to something like `test/asset-compute`. But we can adjust this in a future PR once that has been changed in the cli plugin.